ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -4,3 +4,8 @@
     --pf-c-page__main-section--PaddingLeft: 0;
     --pf-c-page__main-section--PaddingRight: 0;
 }
+
+.pf-topology-resizable-side-bar {
+    /* 155.2px is the height of the page header + nav header */ 
+    height: calc(100vh - 155.2px);
```

Comment:
Open to other suggestions, but seemed like this helped with the scrolling issue

Reply:
I wonder if we could use the PF CCS variables in the calculation for future-proofing?

Reply:
Looks like we can use `var(--pf-c-page__header--MinHeight)` and multiply that by two since we have two levels of headers (nav header + page header). I did notice, however, that it did this


https://user-images.githubusercontent.com/4805485/200041529-b92edf7b-18e1-49f9-b29e-ba14924f88d9.mov

Reply:
Maybe I can add a TODO for considering a different approach